### PR TITLE
add url functions with variables for paths that can be set locally

### PR DESCRIPTION
**Pull request by hippogriffic**
@incuna/frontend please review, from @henrahmagix 's suggestion

### CODE REVIEW

File: incuna-sass/functions/_url-functions.sass [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// The following variables can be overwritten locally to create
```
Comment (henrahmagix) [inline incuna-sass/functions/_url-functions.sass]: `overwritten locally` -> `overwritten in a project`

**Comment by pandalion** [comment]
LGTM!

File: incuna-sass/functions/_url-functions.sass [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+// custom url paths for each function
+
+$relative-image-dir: ../images/ !default
+$relative-font-dir: ../fonts/ !default
+$relative-svg-dir: ../images/svg/ !default
+
+// Set file path for fonts so you can use font-url
+@function font-url($path)
+    @return url('#{relative-font-dir}' + $path)
```
Comment (henrahmagix) [inline incuna-sass/functions/_url-functions.sass]: If using string replacement here, do the same for `$path`
```sass
@return url('#{$relative-foo-dir}#{$path}')
```

Comment (henrahmagix) [inline incuna-sass/functions/_url-functions.sass]: Latter seems simpler to me

**Comment by henrahmagix** [comment]
Just https://github.com/incuna/incuna-sass/pull/78#discussion_r67158165 then 👍

**Comment by hippogriffic** [comment]
@pandalion @henrahmagix fixed

**Comment by grahamgilchrist** [comment]
@henrahmagix FYI if you need to override this per project, you could do this already by just replacing the function.

This does make is slightly easier to just declare a variable though :)

**Comment by hippogriffic** [comment]
yeah that was the idea, I actually thought these were already in here, but @henrahmagix pointed out that they weren't. At least now we can have a single source for the functions themselves and you may not necessarily need to override it every time

we can probably try to figure out what our most common file structure is and use a path to reflect that

**Comment by henrahmagix** [comment]
> This does make is slightly easier to just declare a variable though :)

Also helps with library updates, so projects get the best function =)

**Comment by hippogriffic** [comment]
@incuna/frontend is this good to merge?

**Pull request merged**